### ISSUE TITLE
About the result of PPO

### ISSUE THREAD

**Issue by chen-xihui** [ShengrenHou/DRL-for-Energy-Systems-Optimal-Scheduling#7]
We set the parameters as you mentioned in the paper. The mean reward of 5 episodes is shown as below.
![image](https://github.com/ShengrenHou/DRL-for-Energy-Systems-Optimal-Scheduling/assets/64118984/5ef130e1-b89a-4247-891c-b4e06a44fb01)
As can be seen in the figure, if we train 1000 episodes the mean reward will  approaches -60, which is really bad.
We also train 20000 episodes for each seed. The result can be seen below.
![image](https://github.com/ShengrenHou/DRL-for-Energy-Systems-Optimal-Scheduling/assets/64118984/9949635d-d60b-40dd-98e4-856567efba6d)
The mean reward of 500 episodes finally approaches -20. While the results of training process in your paper is so nice, I wonder what makes the difference? And what are other people's results of the reproduction of the paper？
To be honest, now i want a relatively good results as a baseline. I don't think the mean reward shown above can been seen as a baseline.

**Comment by ShengrenHou** [comment]
Only showing the value of episode reward is not enough to identify the problem, because the magnitude of reward design can influence this. Please show me the detailed operation decision for batteries and generators. Then I can identify whether the performance is good enough. 

I think you can try TD3 or DDPG. Other researchers say that the uploaded version of PPO is not good as good as TD3 in my code. Maybe I uploaded the wrong version CODE. 
But I can not link to my HPC these days, so I still don't have time to carefully recheck it. 
But try more different hyperparameters, like batchsize and learning rates.
